pyproject.toml

```diff
@@ -22,8 +22,8 @@ dependencies = [
     "conda-lock>=3",
     "coverage>=7.10",
     "dagster>=1.11.9",
-    "dagster-dbt>=1!0.27.9",
-    "dagster-postgres>=1!0.27.9",
+    "dagster-dbt", # Needs to follow dagster, but RTD can't deal with epochs
+    "dagster-postgres", # Needs to follow dagster, but RTD can't deal with epochs
```

Comment:
Annoying, RTD doesn't seem to be able to parse the versioning epochs, so this results in an installation error.  However, there are times when Dagster core has been updated while the Dagster libs have not, and without these version specifications it's possible to end up with incompatible versions.